packages/core/bootstrap/src/lib/external-adapter/overrides/presetSymbols.json

```diff
@@ -23,15 +23,9 @@
     "PAX": "USDP"
   },
   "dxfeed": {
-    "AAPL": "AAPL:BFX",
-    "AMZN": "AMZN:BFX",
-    "FB": "FB:BFX",
-    "FTSE": "UKX.IND:TEI",
-    "GOOGL": "GOOGL:BFX",
-    "MSFT": "MSFT:BFX",
-    "N225": "NKY.IND:TEI",
-    "NFLX": "NFLX:BFX",
-    "TSLA": "TSLA:BFX",
```

Comment:
Why are we changing/removing so many of these?

Reply:
I'm happy to take suggestions on how to fix this, but the problem is that these symbol mappings are outdated and result in errors from the api. A lot of the symbols now work without the override.

Reply:
Let's confirm with the data team before making any changes to these. Also, just want you to confirm that the new symbols work with our own dxFeed endpoint (not `https://tools.dxfeed.com`, which is a demo endpoint)

Reply:
Reverted, I needed to use the chainlink endpoint to successfully query these symbols